cpp/README.rst

```diff
@@ -32,6 +32,21 @@ Building and running the code examples
 Additional information geared specifically to understanding the example applications is at
 `AWS SDK for C++ Code Examples <https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/programming-services.html>`_.
 
+
+Docker image (Beta)
+===================
+
+In the near future, this example code will be available in a container image
+hosted on [Amazon Elastic Container Registry (ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html). This image will be pre-loaded
+with all C++ examples with dependencies pre-resolved, allowing you to explore

```

Comment:
Here, and in the other READMEs with this text, recommend the following update:

allowing you to --> so that you can